lib/ansible/modules/cloud/packet/packet_device.py

```diff
@@ -66,22 +68,23 @@
   hostnames:
     description:
       - A hostname of a device, or a list of hostnames.
-      - If given string or one-item list, you can use the C("%d") Python string format to expand numbers from count.
-      - If only one hostname, it might be expanded to list if count>1.
+      - If given string or one-item list, you can use the C("%d") Python string format to expand numbers from I(count).
+      - If only one hostname, it might be expanded to list if I(count)>1.
     aliases: [name]
 
-  lock:
+  locked:
     description:
       - Whether to lock a created device.
     default: false
+    version_added: 2.4
```

Comment:
You can remove this line, you will get a CI failure, though that will go away once this PR is merged